scm-info/org/jboss/spec/javax/resource/_artifact/jboss-connector-api_1.7_spec/scm.yaml

```diff
@@ -0,0 +1,6 @@
+---
+type: "git"
+uri: "https://github.com/jboss/jboss-connector-api_spec.git"
+tagMapping:
+  - pattern: .*
```

Comment:
Is this actually needed? The default algorithm should be able to map this pattern.

Reply:
Yes, because multiple spec versions are found in that repository:

```
java.lang.RuntimeException: Could not determine tag for 1.0.0.Final multiple possible tags were found: [jboss-connector-api_1.7_spec-1.0.0.Final, jboss-connector-api_1.6_spec-1.0.0.Final, jboss-connector-api_1.5_spec-1.0.0.Final]
```

But, what do you think of trying to handle this automatically with something like:

```java
} else if (name.equals(toBuild.getArtifactId() + "-" + version)) {
    //exact match of <artifactId>-<version>
    selectedTag = toBuild.getArtifactId() + "-" + version;
    break;
}
```